File: StraviaTec-Web/Models/Patrocinador.cs
using System;
using System.Collections.Generic;

namespace StraviaTec_Web.Models
{
    public partial class Patrocinador
    {
        public Patrocinador()
        {
            PatrocinadorEvento = new HashSet<PatrocinadorEvento>();
        }

        public int Id { get; set; }
        public string NombreComercial { get; set; }
        public string NombreRepresentante { get; set; }
        public string TelRepresentante { get; set; }
        public string Logo { get; set; }

        public virtual ICollection<PatrocinadorEvento> PatrocinadorEvento { get; set; }
    }
}

File: StraviaTec-Web/ClientApp/src/app/models/patrocinador.ts
export class Patrocinador {
    id: number;
    nombreComercial: string;
    nombreRepresentante: string;
    telRepresentante: string;
    logo: string;

    
    constructor(
        Id: number,
        NombreComercial: string,
        NombreRepresentante: string,
        TelRepresentante: string,
        Logo: string,
    ){
        this.id = Id;
        this.nombreComercial = NombreComercial;
        this.nombreRepresentante = NombreRepresentante;
        this.telRepresentante = TelRepresentante;
        this.logo = Logo;
    }
}

